android/engine/src/main/java/org/smartregister/fhircore/engine/task/FhirTaskExpireUtil.kt

```diff
@@ -62,6 +62,7 @@ constructor(@ApplicationContext val appContext: Context, val defaultRepository:
             { value = of(Task.TaskStatus.RECEIVED.toCoding()) },
           )
 
+          // TODO: This might leave out tasks without an execution period
```

Comment:
true, but isn't that ok, because if they do not have an execution period, then they are not past their expiry, so they would have been filtered out below anyhow

Reply:
That's a good point. This assumes that all Tasks with a `Task.restriction` period also have a `Task.executionPeriod`. I believe this was the initial scoping for task expiry

Reply:
I will remove this and we can create another issue once we decide on this